### DISCUSSION TITLE
shutdown SO 2.3.0 gracefully

### DISCUSSION THREAD

**Discussion by four80eastfan** [Security-Onion-Solutions/securityonion#1619] [Unsupported Versions] [answered]
what's the best way to shutdown Security Onion 2.3.0 gracefully? i used to run 'so-stop' and then 'shutdown -h now' on 16.04 (not sure if that was the best way). 'so-stop' no longer seems to be an option in 2.3.0. i see that i can run 'so-stop kibana' etc. but is there a script that stops them all? if not, what order, if any, should i stop the services in? thanks

**Comment by TOoSmOotH** [comment, marked as answer]
You should be able to do a shutdown -h now and it will bring everything down properly.
